src/app/network/crypto.ts

```diff
@@ -139,51 +140,40 @@ export async function getEncryptedFile(
     const status = await readable.read();
 
     if (!status.done) {
-      hasher.process(status.value);
+      hasher.update(status.value);
```

Comment:
Did you see any improvements in performance @TamaraFinogina ? I assume the answer it's yes by the WASM usage, it would be nice to highlight that on the PR description.

Reply:
How can I test it?